src/jobs/wallets/fetch-history-queue.ts

```diff
@@ -18,7 +18,7 @@ export const queue = new Queue(QUEUE_NAME, {
     // any failed processes to be done by subsequent jobs
     removeOnComplete: true,
     //todo: will make it true, when have fallback mechanism
-    removeOnFail: false,
+    removeOnFail: { age: 24 * 3600 },
```

Comment:
Since this value (24 * 3600) is being repeated in multiple places, please convert this to a config variable.